app/services/datastream_downloader.rb

```diff
@@ -10,6 +10,7 @@ def default_supported_ssgs
     ::SupportedSsg.all
                   .sort_by(&:comparable_version)
                   .reverse
+                  .uniq { |ssg| [ssg.version, ssg.os_major_version] }
```

Comment:
How is it with the determinism of the uniqueness and sorting? It is sorted by a `comparable_version` based off of `version` which for some packages would the same.

```
scap-security-guide-0.1.57-3.el8_4: 0.1.57
scap-security-guide-0.1.57-5.el8: 0.1.57
```

Reply:
@vkrizan adjusted